src/ahbicht/expressions/expression_resolver.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module makes it possible to parse expressions with all its subexpressions
+for example ahb_expressions with condition_expressions or condition_expressions with packages.
```

Comment:
```suggestion
for example ahb_expressions with nested condition_expressions or condition_expressions with nested packages.
```
ich weiß nicht ob "nested" das beste wort ist aber ich fände es gut wenn das verhältnis der beiden expressions klar wird (also das eine die andere enthält).

Reply:
Wie wäre es mit:
```suggestion
for example ahb_expressions which contain condition_expressions or condition_expressions which contain packages.
```

Reply:
ja, gerne 👍